adoc/code/algorithms.cpp

```diff
@@ -31,5 +31,6 @@ myQueue.submit([&](handler& cgh) {
   });
 });
 
-assert(outputBuf.get_host_access()[0] == 523776 &&
-       outputBuf.get_host_access()[1] == 120);
+auto a = outputBuf.get_host_access();
+assert(a[0] == 523776 &&
+       a[1] == 120);
```

Comment:
This is a good catch!  Since we are changing this code anyway, I think the following might be a bit cleaner:

```
host_accessor a { outputBuf };
assert(a[0] == 523776 && a[1] == 120);
```

I think we generally prefer to show examples that use the CTAD versions of the accessor constructors rather than the `get_host_access` factory functions.  Note that we use the CTAD version earlier in this same example.

Reply:
Yea, that makes sense. Made this change.